examples/eks-cluster-with-self-managed-node-groups/main.tf

```diff
@@ -121,7 +121,14 @@ module "aws-eks-accelerator-for-terraform" {
             systemctl enable amazon-ssm-agent && systemctl start amazon-ssm-agent \
         EOT
 
-      disk_size     = 20
+      block_device_mappings = [
```

Comment:
Add multiple device blocks for example with instance type that uses NVMe

Reply:
I added three block devices that will result in three disks attached into the instance: one at /, another at /local1, and the last one at /local2.
Regarding instances that have local NVMes, I added that at docs/node-group.md.

Reply:
Done.